erdpy/myprocess.py

```diff
@@ -15,7 +15,7 @@ def run_process(args: List[str], env: Any = None):
     try:
         output = subprocess.check_output(args, shell=False, universal_newlines=True, stderr=subprocess.STDOUT, env=env)
         logger.info("Successful run. Output:")
-        print(output or "[No output]")
+        # print(output or "[No output]")
```

Comment:
Please uncomment - important for the other components that call the function (such as the `contract build` commands). Or perhaps add an extra parameter to specify to `dump_to_stdout` or not, with default to dump.

Reply:
done.